commands.js

```diff
@@ -1736,6 +1736,36 @@ exports.commands = {
 	},
 	modchathelp: ["/modchat [off/autoconfirmed/+/%/@/*/#/&/~] - Set the level of moderated chat. Requires: *, @ for off/autoconfirmed/+ options, # & ~ for all the options"],
 
+	slowchat: function (target, room, user) {
+		if (!target) return this.sendReply("Slow chat is currently set to: " + (room.slowchat ? room.slowchat : false));
+		if (target === 'help') return this.parse("/help slowchat");
+		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (!this.can('slowchat', null, room)) return false;
+
+		let slowchatRange = [2, 60];
+		if (target === 'off' || target === 'disable') {
+			if (!room.slowchat) return this.errorReply("Slow chat is already disabled in this room.");
+			room.slowchat = false;
+			this.add("|raw|<div class=\"broadcast-blue\"><b>Slow chat was disabled!</b><br />There is no longer a set minimum time between messages.</div>");
+		} else if (parseInt(target)) {
+			if (room.slowchat === parseInt(target)) return this.errorReply("Slow chat is already set for " + room.slowchat + " seconds in this room.");
```

Comment:
You use `parseInt(target)` like fifty times here, just make a temporary variable for it.